aggregator/src/blob_consistency.rs

```diff
@@ -0,0 +1,27 @@
+use cfg_if::cfg_if;
+
+// enum DataAvailability {
+//     Eip4844,
+//     Avail,
+// }
+
+mod blob_data;
+use blob_data::{AssignedBlobDataExport, BlobDataConfig};
+
+// TODO: remove dead code instead
+#[allow(dead_code)]
+mod avail;
+
+// TODO: remove dead code instead
+#[allow(dead_code)]
+mod eip4844;
+
+cfg_if! {
+    if #[cfg(feature = "da-avail")] {
+        // const DATA_AVAILABILITY: DataAvailability = DataAvailability::Avail;
```

Comment:
Perhaps this const can be defined in the submodule (in `avail` and `eip4844`) and then exported.

Reply:
Thanks, I will keep this in mind when this line gets uncommented.